tornado-api/src/main/java/uk/ac/manchester/tornado/api/TaskGraph.java

```diff
@@ -909,4 +909,8 @@ long getCurrentDeviceMemoryUsage() {
     void mapOnDeviceMemoryRegion(Object destArray, Object srcArray, long offset, TornadoTaskGraphInterface taskGraphSrc) {
         taskGraphImpl.mapOnDeviceMemoryRegion(destArray, srcArray, offset, taskGraphSrc);
     }
+
+    public TornadoTaskGraphInterface getTaskGraphImpl() {
```

Comment:
Same with this method. The immutable TaskGraph is only used in the ExecutionPlan